upstream/CVE-2019-11251.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "CVE-2019-11251",
+  "modified": "2020-02-03T15:12:22Z",
+  "published": "2020-02-03T15:12:22Z",
+  "summary": "kubectl cp symlink vulnerability",
+  "details": "The Kubernetes kubectl cp command in versions 1.1-1.12, and versions prior to 1.13.11, 1.14.7, and 1.15.4 allows a combination of two symlinks provided by tar output of a malicious container to place a file outside of the destination directory specified in the kubectl cp invocation. This could be used to allow an attacker to place a nefarious file using a symlink, outside of the destination tree.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubectl"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:N/I:H/A:N"
```

Comment:
not sure if it make a different however cve reported on [github](https://github.com/advisories/GHSA-6qfg-8799-r575) is :                  `CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:N/I:H/A:N`
where cve here is : `CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:N/I:H/A:N`

Reply:
It's rated by GitHub, not Kubernetes.
https://github.com/advisories/GHSA-6qfg-8799-r575